backend/app/utils/encodings/encodings_list.py

```diff
@@ -0,0 +1,97 @@
+encoding_list = ['ascii',
```

Comment:
Use a frozenset or tuple if this is immutable, as it should be.

Reply:
Yes, it is immutable so I am using a frozenset now.